src/store/wizardSlice.ts

```diff
@@ -40,6 +42,13 @@ type wizardState = {
     shareMethod: AwsShareMethod;
     source: V1ListSourceResponseItem | undefined;
   };
+  azure: {
+    shareMethod: AzureShareMethod;
+    tenantId: string;
+    subscriptionId: string;
+    source: AzureUploadRequestOptions['source_id'] | undefined;
```

Comment:
Let’s change the type here to just be `string`. 

If you actually look at the type definition for `AzureUploadRequestOptions['source_id']` it is `string | undefined`. So what is written here is actually redundant (as undefined appears twice if you expand out the union) and also, we don’t want this to ever be `undefined` since we will just use `’’` as the empty state.